### DISCUSSION TITLE
What are the statuses for a Bitbucket pipeline?

### DISCUSSION THREAD

**Discussion by ArcherEmiya05** [badges/shields#10095] [General] [answered]
Hi, I am trying to use Shields IO to generate badge for build status. So far the statuses I was able to generate is `passing`, `never built` and `not found` for private repo, I cannot find yet a public repo that has latest built on its pipeline that is failed to see what will be the badge looks like. Is there a list of statuses that I can look at? Tried looking at the docs but no luck. Thanks!

**Comment by chris48s** [comment, marked as answer]
These are the values we recognise as pipeline statuses from BitBucket
https://github.com/badges/shields/blob/d8e3452fd2ed9d4f04f89c31972f5c3dba257497/services/bitbucket/bitbucket-pipelines.service.js#L13-L17
